src/framework/xr/xr-plane.js

```diff
@@ -150,24 +152,36 @@ class XrPlane extends EventHandler {
     }
 
     /**
-     * Plane's specific orientation (horizontal or vertical) or null if orientation is anything else.
+     * Gets the plane's specific orientation. This can be "horizontal" for planes that are parallel
+     * to the ground, "vertical" for planes that are perpendicular to the ground, or `null` if the
+     * orientation is different or unknown.
      *
      * @type {string|null}
+     * @example
```

Comment:
do we want to list the possible strings in the `@type` tag instead of `string`?

Reply:
Done.